### PR TITLE
Footer update - Feature/36086

### PR DESCRIPTION
**Pull request by anqaka**
#26

### CODE REVIEW

File: Magento_Theme/templates/footer/footer-links.phtml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        <li class="
+                dropdown-list__item
+                dropdown-list__item--collapse
+            "
+        >
+            <button class="dropdown-list__label "
+                    aria-expanded="false"
+                    aria-controls="dropdown-alpaca"
+                    data-dropdown="dropdown-alpaca"
+            >
```
Comment (szafran89) [inline Magento_Theme/templates/footer/footer-links.phtml]: missing button type

File: Magento_Theme/templates/footer/footer-links.phtml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+     "
+     data-mage-init='{ "dropdownList": {} }'
+>
+    <ul class="dropdown-list__list">
+        <li class="
+                dropdown-list__item
+                dropdown-list__item--collapse
+            "
+        >
+            <button class="dropdown-list__label "
```
Comment (szafran89) [inline Magento_Theme/templates/footer/footer-links.phtml]: useless space

File: Magento_Theme/templates/footer/footer-links.phtml (hunk 1 above)
Comment (szafran89) [reply Magento_Theme/templates/footer/footer-links.phtml]: Did you add type? I can't see. I think should be better to add.

Comment (anqaka) [reply Magento_Theme/templates/footer/footer-links.phtml]: You're right, now added!

File: Magento_Theme/templates/footer/footer-links.phtml (hunk 2 above)
Comment (anqaka) [reply Magento_Theme/templates/footer/footer-links.phtml]: corrected

**Pull request merged**